### PR TITLE
Spctr 2058 Fix: Icon List - Resolved an issue as the link was getting disabled for the icon list child.

### PR DESCRIPTION
**Pull request by jamyrbsf**
### Description
https://brainstormforce.atlassian.net/browse/SPCTR-2058

### Screenshots
https://share.getcloudapp.com/WnuZ2vdj
https://share.getcloudapp.com/4gu4QqbG
https://share.getcloudapp.com/4gu4Qq79

### Types of changes
Fixed links working issue on frontend

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/icon-list-child/edit.js [hunk 1]
```diff
@@ -20,23 +20,6 @@ const UAGBIconListChild = ( props ) => {
...
 	const deviceType = useDeviceType();
 
-	useEffect( () => {
-		const {
-			disableLink
-		} = props.attributes;
-		if ( 'yes' === uagb_blocks_info.uagb_old_user_less_than_2 ) {
-			if( ! disableLink ) {
-				props.setAttributes( { disableLink: true } );
-			} else {
```
Comment (RajkiranBagal) [inline src/blocks/icon-list-child/edit.js:30]: Why did we remove this code?

File: src/blocks/icon-list-child/render.js [hunk 2]
```diff
@@ -66,12 +66,6 @@ const Render = ( props ) => {
 	const targetVal = target ? '_blank' : '_self';
 	const linkUrl = disableLink ? link : '/';
 
-	let disableLinks = disableLink;
-
```
Comment (RajkiranBagal) [inline src/blocks/icon-list-child/render.js:70]: Why did we remove this code?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/icon-list-child/edit.js (hunk 1 above)
Comment (jamyrbsf) [reply src/blocks/icon-list-child/edit.js:30]: It was causing the links to be disabled when the toggle is enabled. So I removed it

File: src/blocks/icon-list-child/render.js (hunk 2 above)
Comment (jamyrbsf) [reply src/blocks/icon-list-child/render.js:70]: This value was not used in my new code so I removed it.

File: src/blocks/icon-list-child/edit.js (hunk 1 above)
Comment (RajkiranBagal) [reply src/blocks/icon-list-child/edit.js:30]: Did you check the case why we added that code in first place? Also you have removed the code in which we sset block_id which is unrelated here.

Comment (RajkiranBagal) [reply src/blocks/icon-list-child/edit.js:30]: Why we removed this check? uagb_old_user_less_than_2? It must be added for some reason, did we check that?

Comment (jamyrbsf) [reply src/blocks/icon-list-child/edit.js:30]: As we discussed in the call, we have used single variables to maintain 2 values which are causing some problems. sol I removed it.

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**